baram/ec2_manager.py

```diff
@@ -1,60 +1,252 @@
 import boto3
+import botocore.exceptions
+
+from baram.log_manager import LogManager
 
 
 class EC2Manager(object):
     def __init__(self):
         self.cli = boto3.client('ec2')
+        self.logger = LogManager.get_logger()
 
     def list_security_groups(self):
-        '''
+        """
```

Comment:
Do you need to change comment separator?

Reply:
PyCharm said `Triple double-quoted strings should be used for docstrings.`